fiftyone/utils/cvat.py

```diff
@@ -2498,6 +2498,8 @@ class CVATBackendConfig(foua.AnnotationBackendConfig):
         password (None): the CVAT password
         headers (None): an optional dict of headers to add to all CVAT API
             requests
+        task_size (None): an optional maximum number of images to upload per
```

Comment:
FYI- I moved `task_size` up here so that it appears near other relevant parameters.

I think this is more user-friendly than strictly adding new kwargs to the end, which makes it harder to grok the available options. This is also why I *always* recommend using `key=value` syntax when calling any function with kwargs, so that order can be changed later in cases like this.